team/index.html

```diff
@@ -588,7 +588,20 @@ <h1>Developers</h1>
 						<span>PDX Git Engineer</span>
 					</div>
 				</div>
-
+				<div class="col-md-3 col-sm-6 speaker-column">
+					<div class="speaker">
+						<div class="image-holder">
+							<img class="background-image" alt="Neel Trivedi" src="../img/team/NeelTrivedi.jpg">
+							<div class="hover-state text-center preserve3d">
+								<div class="social-links vertical-align">
+									<a href="https://github.com/neel1998"><i class="icon fa fa-github"></i></a>
```

Comment:
```suggestion
									<a href="https://github.com/neel1998" target="blank"><i class="icon fa fa-github"></i></a>
```

Reply:
OK i changed sentence and added img/team/VinaySajjanapu.jpg photo